import please.svn
#please.svn.delete_problem('a11')
print(please.svn.svn_problem_exists('a1_gurovic'))
